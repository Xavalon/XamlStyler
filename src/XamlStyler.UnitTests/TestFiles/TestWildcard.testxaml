<Window x:Class="MainWindow"
        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
    <Grid>
        <Button
            Width="200"
            Height="75"
            AutomationProperties.AutomationId="TestButton"
            AutomationProperties.HelpText="Test Button"
            Content="Test Button"
            IsEnabled="True"
            ToolTipService.Placement="Bottom"
            ToolTipService.ToolTip="Test"
            Grid.Row="1"
            Grid.Column="1"
            Visibility="Visible" />
    </Grid>
</Window>
