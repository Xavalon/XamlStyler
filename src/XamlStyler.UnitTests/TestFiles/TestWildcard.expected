<Window x:Class="MainWindow"
        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
  <Grid>
    <Button Grid.Column="1"
            Grid.Row="1"
            Width="200"
            Height="75"
            Content="Test Button"
            IsEnabled="True"
            Visibility="Visible"
            ToolTipService.Placement="Bottom"
            ToolTipService.ToolTip="Test"
            AutomationProperties.AutomationId="TestButton"
            AutomationProperties.HelpText="Test Button" />
  </Grid>
</Window>
